packages/hero/index.tsx

```diff
@@ -39,7 +39,7 @@ const Hero = ({
               <InlineVideo
                 solution={solution}
                 url={url}
-                gradientPosition={false}
```

Comment:
It should be in the latest release https://github.com/hashicorp/react-components/blob/main/packages/inline-video/types.ts#L6

Reply:
Yup - hadn't updated the package. Should be good here now!